python/tk_maya_playblast/playblast.py

```diff
@@ -103,34 +111,49 @@ def __createPlayblast(self, **overridePlayblastParams):
         result = self._app.execute_hook("hook_post_playblast", action="copy_file", data=localPlayblastPath)
         
         if result:
-            self._app.log_info("Playblast local file created: %s" % localPlayblastPath)
-
-        # register new Version entity in shotgun or update existing version, minimize shotgun data
-        playblast_movie = self.shotPlayblastPath
-        project = self._app.context.project
-        entity = self._app.context.entity
-        task = self._app.context.task
-
-        data = { 'project': project,
-                 'code': os.path.basename(playblast_movie),
-                 'description': 'automatic generated by playblast app',
-                 'sg_path_to_movie': playblast_movie,
-                 'entity': entity,
-                 'sg_task': task,
-               }
-        self._app.log_debug("Version-creation hook data:\n" + pprint.pformat(data))
-        result = self._app.execute_hook("hook_post_playblast", action="create_version", data=data)
-        self._app.log_debug("Version-creation hook result:\n" + pprint.pformat(result))
-
-        # upload QT file if creation or update process run succesfully
-        if result and self.__uploadToShotgun:
-            result = self._app.execute_hook("hook_post_playblast", action="upload_movie",
-                                            data=dict(path=data["sg_path_to_movie"],
-                                                      project=project,
-                                                      version_id=result["id"]))
+            self._app.log_info("Playblast local file created: %s" % result)
+
+        if self.__showViewer and viewer_issues:
+            self._app.log_info("Opening RV...")
```

Comment:
Try use the newer `logger.info` as well as standard logging `%s` substitution. 

There's more throughout the PR but here are just one of the cases

```suggestion
            self._app.logger.info("Playblast local file created: %s", result)

        if self.__showViewer and viewer_issues:
            self._app.logger.info("Opening RV...")
```